llvm/test/CodeGen/RISCV/GlobalISel/instruction-select/ptrmask-rv32.mir

```diff
@@ -0,0 +1,25 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
```

Comment:
Should we test the ANDI case which is probably the more common use case for G_PTRMASK?